README.md

```diff
@@ -2,42 +2,38 @@
 | **Installation** | [![Releases](https://img.shields.io/badge/obtain-latest-green.svg)](https://github.com/openforcefield/openff-toolkit/releases) [![Conda](https://img.shields.io/conda/v/conda-forge/openff-toolkit.svg)](https://anaconda.org/conda-forge/openff-toolkit) [![Last updated](https://anaconda.org/conda-forge/openff-toolkit/badges/latest_release_relative_date.svg)](https://anaconda.org/conda-forge/openff-toolkit) [![Anaconda Cloud downloads](https://anaconda.org/conda-forge/openff-toolkit/badges/downloads.svg)](https://anaconda.org/conda-forge/openff-toolkit)          |
 
 # The Open Force Field toolkit
 
 The Open Force Field Toolkit, built by the [Open Force Field Initiative](http://openforcefield.org), is a Python toolkit for the development and application of modern molecular mechanics force fields based on direct chemical perception and rigorous statistical parameterization methods.
 
 The toolkit currently covers two main areas we have committed to stably maintain throughout their lifetimes:
-* Tools for using the [SMIRKS Native Open Force Field (SMIRNOFF) specification](https://openforcefield.github.io/standards/standards/smirnoff/)
+* Tools for using [SMIRKS Native Open Force Field (SMIRNOFF)](https://openforcefield.github.io/standards/standards/smirnoff/) force fields
 * Tools for [direct chemical environment perception](https://dx.doi.org/10.1021/acs.jctc.8b00640) and manipulation
 
 **Note**: Prior to version 0.9.0, this toolkit and its associated repository were named `openforcefield` and used different import paths. For details on this change and migration instructions, see the [release notes](https://open-forcefield-toolkit.readthedocs.io/en/stable/releasehistory.html#namespace-migration) of version 0.9.0.
 
 ## Documentation
 
-[Documentation](https://open-forcefield-toolkit.readthedocs.io/en/stable/) for the Open Force Field Toolkit is hosted at [readthedocs](https://open-forcefield-toolkit.readthedocs.io/en/stable). Example notebooks are hosted on available in the `examples/` repository.
+[Documentation](https://open-forcefield-toolkit.readthedocs.io/en/stable/) for the Open Force Field Toolkit is hosted at [readthedocs](https://open-forcefield-toolkit.readthedocs.io/en/stable). Example notebooks are available in the `examples/` repository and also hosted on the [`Open Force Field`](https://docs.openforcefield.org/en/latest/examples.html) website.
```

Comment:
This links to the ecosystem docs, not just the toolkit-specific ones, but I think it's appropriate in the sense that the toolkit's README is a common first landing page for people getting started with the infrastructure as a whole